sympy/integrals/rubi/utility_function.py

```diff
@@ -2710,7 +2710,7 @@ def PolynomialTermQ(u, x):
     a = Wild('a', exclude=[x])
     n = Wild('n', exclude=[x])
     Match = u.match(a*x**n)
-    if Match and IntegerQ(Match[n]) and Match[n]>0:
+    if Match and IntegerQ(Match[n]) and Greater(Match[n], S(0)):
```

Comment:
No, `Greater` is implemented such that it returns `False` if the inequality is `Relational`.